<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerwalteMedienDialog</class>
 <widget class="QDialog" name="VerwalteMedienDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Medien</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="buecher">
    <attribute name="title">
     <string>BÃ¼cher</string>
    </attribute>
    <widget class="QTableWidget" name="buecherTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="filme">
    <attribute name="title">
     <string>Filme</string>
    </attribute>
    <widget class="QTableWidget" name="filmeTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VerwalteMedienDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VerwalteMedienDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
